### PR TITLE
Added disabled property to combobox.

### PR DESCRIPTION
**Pull request by jermsv1**
[Review on Reviewable](https://reviewable.io/reviews/vaadin/vaadin-combo-box/143)

### CODE REVIEW

**Comment by Saulis** [comment]
Hi there! If you would be kind enough to sign the CLA, we would be happy to take this PR in.

**Comment by jounik** [comment]
Don’t mean to be a party pooper, but we will need to add tests for this also.

Does the element already prevent the dropdown from opening when the element is disabled?

A bit off topic, but it seems weird that paper-input still allows you to focus the field when it’s disabled. I would expect to be able to skip the field with the tab key in a form if it is disabled.

**Comment by jounik** [comment]
Forgot to say – like Sauli mentioned – thank you for the PR, we’d be happy to take it!

File: vaadin-dropdown-behavior.html [hunk 1]
```diff
@@ -119,4 +124,4 @@
       }
     }
   };
-</script>
+</script>
```
Comment (jounik) [inline vaadin-dropdown-behavior.html]: Minor nitpick: unnecessary white space change.

**Comment by Saulis** [comment]
@samiheikki and myself will be helping out by adding tests and other missing pieces.

**Comment by jermsv1** [comment]
Thank you! I'm a little tied up with another project and won't get to it
till later next week

**Comment by Saulis** [comment]
We've moved the changes to PR #149

**Pull request closed without merging**